app/controllers/applications_controller.rb

```diff
@@ -1,4 +1,22 @@
 class ApplicationsController < ApplicationController
+  def new
+  end
+
+  def create
+    application = Application.create({
```

Comment:
since we are creating an object from AR, we don't need to assign to variable.
Also prefer to use `.create!` to cause crashes as early as possible